ZukoAzula2016/src/org/team1540/zukoazula/IntakeArm.java

```diff
@@ -17,6 +19,13 @@
     private static final FloatInput intakeArmAxis = ZukoAzula.controlBinding.addFloat("Intake Arm Axis").deadzone(0.2f).negated();
 
     public static void setup() throws ExtendedMotorFailureException {
-        intakeArmAxis.send(intakeArmCAN.simpleControl());
+        FloatCell armLow = ZukoAzula.mainTuning.getFloat("Intake Arm Low Position", 0f);
+        FloatCell armHigh = ZukoAzula.mainTuning.getFloat("Intake Arm High Position", 1f);
+        FloatInput armPosition = encoder.normalize(armLow, armHigh);
+        BooleanInput stop = armPosition.atLeast(1).and(intakeArmAxis.atLeast(0)).or(armPosition.atMost(0).and(intakeArmAxis.atMost(0)));
```

Comment:
More explanation. Perhaps via variable names (and therefore multiple lines), perhaps via comments.